meta-lmp-base/classes/uboot-fitimage.bbclass

```diff
@@ -45,6 +47,7 @@ uboot_fitimage_assemble() {
 	ubootloadaddr=${2}
 	opteeloadaddr=${3}
 	atfloadaddr=${4}
+	bootscrloadaddr=${5}
```

Comment:
We should also make a condition to see if this variable is indeed valid (set by the board). Otherwise BOOTSCR_SUPPORT can end up being set to true (based on u-boot-ostree-scr-fit support), but missing BOOTSCR_LOAD_ADDR, resulting in an invalid fit.

Reply:
Or we could extend the BOOTSCR_SUPPORT variable set to check for 2 conditions, PREFERRED_PROVIDER_u-boot-default-script and BOOTSCR_LOAD_ADDR not being empty.

Reply:
fixed